src/libraries/System.Runtime/tests/System.Reflection.Tests/AssemblyTests.cs

```diff
@@ -181,6 +181,19 @@ public void GetEntryAssembly()
             Assert.True(correct, $"Unexpected assembly name {assembly}");
         }
 
+        [Fact]
+        public void SetEntryAssembly()
```

Comment:
This test may want to use `RemoteExecutor.Invoke(` to avoid interfering with the existing GetEntryAssembly test.